workflows/streptococcus_pyogenes.nf

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl=2
+
+include { get_meta                                  } from '../methods/get_meta.nf'
+include { amrfinderplus                             } from '../nextflow-modules/modules/amrfinderplus/main.nf'
+include { bracken                                   } from '../nextflow-modules/modules/bracken/main.nf'
+include { bwa_index                                 } from '../nextflow-modules/modules/bwa/main.nf'
+include { bwa_mem as bwa_mem_dedup                  } from '../nextflow-modules/modules/bwa/main.nf'
+include { chewbbaca_allelecall                      } from '../nextflow-modules/modules/chewbbaca/main.nf'
+include { chewbbaca_create_batch_list               } from '../nextflow-modules/modules/chewbbaca/main.nf'
+include { chewbbaca_split_results                   } from '../nextflow-modules/modules/chewbbaca/main.nf'
+include { copy_to_cron                              } from '../nextflow-modules/modules/cron/main.nf'
+include { create_analysis_result                    } from '../nextflow-modules/modules/prp/main.nf'
+include { create_cdm_input                          } from '../nextflow-modules/modules/prp/main.nf'
+include { create_yaml                               } from '../nextflow-modules/modules/yaml/main.nf'
+include { emmtyper                                  } from '../nextflow-modules/modules/emmtyper/main.nf'
+include { export_to_cdm                             } from '../nextflow-modules/modules/cmd/main.nf'
+include { freebayes                                 } from '../nextflow-modules/modules/freebayes/main.nf'
+include { kraken                                    } from '../nextflow-modules/modules/kraken/main.nf'
+include { mask_polymorph_assembly                   } from '../nextflow-modules/modules/mask/main.nf'
+include { mlst                                      } from '../nextflow-modules/modules/mlst/main.nf'
+include { resfinder                                 } from '../nextflow-modules/modules/resfinder/main.nf'
+include { samtools_index as samtools_index_assembly } from '../nextflow-modules/modules/samtools/main.nf'
+include { virulencefinder                           } from '../nextflow-modules/modules/virulencefinder/main.nf'
+include { CALL_BACTERIAL_BASE                       } from '../workflows/bacterial_base.nf'
+
+workflow CALL_STREPTOCOCCUS_PYOGENES {
+    Channel.fromPath(params.csv).splitCsv(header:true)
+        .map{ row -> get_meta(row) }
+        .branch {
+        iontorrent: it[2] == "iontorrent"
```

Comment:
Same here: Would be prettier (and a tad bit more readable) with an extra indentation step here.